### ISSUE TITLE
Integrate test coverage and static code analysis report

### ISSUE THREAD

**Issue by pecko** [debrief/limpet#224]
We should try to integrate these reports. 
See #157 and #158

**Comment by IanMayo** [comment]
We have two sets of code quality reports: 
- Findbugs/CPD/PMD/CheckStyle - in **Maven**
- Jacoco (Java Code Coverage)  - in **Ant**

It would be easier to navigate these reports if they were combined. Here is a Maven task for JaCoCo:
http://eclemma.org/jacoco/trunk/doc/maven.html

Hopefully using it will combine the two sets of reports.

**Comment by pecko** [comment]
I have created the PR: https://github.com/debrief/limpet/pull/231
It adds coverage link to code analysis reports: http://debrief.github.io/limpet-update/static/report/

I haven't manage to create the coverage reports with maven plugin and tycho, but maven coverage reports are the same as ant coverage reports.
We can polish main index.html (info.limpet.site/index.html). I also can try to find the way to generate it with maven, but, IMO, it still will be the same.

I have merged the PR in order to test links. 

Screenshot (http://debrief.github.io/limpet-update/static/report/):
![reports](https://cloud.githubusercontent.com/assets/534584/12158238/8066e7bc-b4d8-11e5-9c90-db76c2463f50.png)
